nailgun/entities.py

```diff
@@ -4827,6 +4836,144 @@ def upload_content(self, synchronous=True, **kwargs):
             )
         return json
 
+    def content_upload_id(self, synchronous=True, **kwargs):
+        """Create an upload request
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all JSON decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.post(self.path('content_uploads'), **kwargs)
+        json = _handle_response(response, self._server_config, synchronous)
+        return json
+
+    def content_upload_put(self, upload_id, filepath, **kwargs):
+        """Upload content chunked into a repository
+
+        Expects an upload_id and a path to a file
```

Comment:
Can you add the corresponding docstring tokens for the required arguments and their types?